plugins/excel/core/src/main/java/org/pentaho/di/trans/steps/excelwriter/ExcelWriterStepMeta.java

```diff
@@ -725,6 +776,11 @@ private void readData( Node stepnode ) throws KettleXMLException {
     }
   }
 
+  private boolean getBooleanValue( Node node, String tag, boolean defaultValue ) {
+    String value = XMLHandler.getTagValue( node, tag );
+    return Optional.ofNullable( value ).filter( StringUtils::isBlank ).map( val -> "Y".equals( val ) ).orElse( false );
```

Comment:
filtering to include only blank values? 
The way I read this, it can only ever return false.